### ISSUE TITLE
the accuracy is lower than 72.2% in HMDB51-1

### ISSUE THREAD

**Issue by runtoforever** [craston/MARS#35]
Hello, I want to kown why my validation accuracy is lower than 72.2%, it's about 71.3%.  I set batch size as 125 and else didn't change. from now as I known, batch size maybe won't influence the result .Thanks in advance.

**Comment by craston** [comment]
Did you calculate the validation accuracy per clip or over the entire video? The clip accuracy would be lower that the video accuracy (averaged clip accuracy) reported in the paper

**Comment by runtoforever** [comment]
I used your trained model to reproduce on testset, I am very happy to get one about the same accuracy reported in your paper. I want to confirm a question ahout HMDB51 train/test dataset. According to your code, I realized you use testset as validationset and testset and The accuracy reported in the paper is on testset. eg. Running the trained model again on the test set gets the accuracy.



At 2021-01-05 21:46:56, "craston" <notifications@github.com> wrote:

Did you calculate the validation accuracy per clip or over the entire video? The clip accuracy would be lower that the video accuracy (averaged clip accuracy) reported in the paper

**Comment by craston** [comment]
Yes I used the test set for validation

**Comment by runtoforever** [comment]
Thank you. I understand and still have another quetion. I try to train model on UCF101 dataset on pretrained model kinetics400. The train accuracy is very high up to 98% but the validation accuracy is lower only to 80%. I think this is over-fitting but don't kown how to solve it. could you give me some advice to deal with?



在 2021-01-06 22:12:20，"craston" <notifications@github.com> 写道：

Yes I used the test set for validation

**Comment by craston** [comment]
Yes, there's a overfitting with UCF101/HMDB51 datasets as they are very small datasets to train a large model like ResNeXt101. Therefore the model will overfit the limited training data. I haven't looked how to solve this; however you could try lowering the learning rate. However, I still think it will lead to overfitting.

**Comment by runtoforever** [comment]
Thank you. I have solved this problem. I trained too many parameters before so that the model will overfit the trainding data. When I only trained the last module the validation accuracy increased.



At 2021-01-11 16:28:50, "craston" <notifications@github.com> wrote:

Yes, there's a overfitting with UCF101/HMDB51 datasets as they are very small datasets to train a large model like ResNeXt101. Therefore the model will overfit the limited training data. I haven't looked how to solve this; however you could try lowering the learning rate. However, I still think it will lead to overfitting.

**Comment by runtoforever** [comment]
hello，recently I tested it on UCF101 dataset with Following your Settings, and the accuracy reached 93.9% but the accuracy in the paper was 94.6%. What can I do to improve the accuracy? Thanks.



At 2021-01-12 09:42:56, "王雷" <nohesitate2019@163.com> wrote:

Thank you. I have solved this problem. I trained too many parameters before so that the model will overfit the trainding data. When I only trained the last module the validation accuracy increased.



At 2021-01-11 16:28:50, "craston" <notifications@github.com> wrote:

Yes, there's a overfitting with UCF101/HMDB51 datasets as they are very small datasets to train a large model like ResNeXt101. Therefore the model will overfit the limited training data. I haven't looked how to solve this; however you could try lowering the learning rate. However, I still think it will lead to overfitting.

**Comment by HuangZuShu** [comment]
> Thank you. I have solved this problem. I trained too many parameters before so that the model will overfit the trainding data. When I only trained the last module the validation accuracy increased. At 2021-01-11 16:28:50, "craston" <notifications@github.com> wrote: Yes, there's a overfitting with UCF101/HMDB51 datasets as they are very small datasets to train a large model like ResNeXt101. Therefore the model will overfit the limited training data. I haven't looked how to solve this; however you could try lowering the learning rate. However, I still think it will lead to overfitting. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Hi， I want to know how do you trian the model with less parameters. Would you mind giving more details?

**Comment by HuangZuShu** [comment]
> Thank you. I have solved this problem. I trained too many parameters before so that the model will overfit the trainding data. When I only trained the last module the validation accuracy increased. At 2021-01-11 16:28:50, "craston" <notifications@github.com> wrote: Yes, there's a overfitting with UCF101/HMDB51 datasets as they are very small datasets to train a large model like ResNeXt101. Therefore the model will overfit the limited training data. I haven't looked how to solve this; however you could try lowering the learning rate. However, I still think it will lead to overfitting. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

@runtoforever 
Hi， I want to know how do you trian the model with less parameters. Would you mind giving more details?

**Comment by runtoforever** [comment]
Thank you. I have solved it. The accuracy is low because I trained the model with the parameters of last block and fc layer



At 2021-03-16 09:05:33, "hzs" ***@***.***> wrote:

Thank you. I have solved this problem. I trained too many parameters before so that the model will overfit the trainding data. When I only trained the last module the validation accuracy increased. At 2021-01-11 16:28:50, "craston" ***@***.*** wrote: Yes, there's a overfitting with UCF101/HMDB51 datasets as they are very small datasets to train a large model like ResNeXt101. Therefore the model will overfit the limited training data. I haven't looked how to solve this; however you could try lowering the learning rate. However, I still think it will lead to overfitting. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

@runtoforever
Hi， I want to know how do you trian the model with less parameters. Would you mind giving more details?
